lib/orderbook/OrderBook.ts

```diff
@@ -232,18 +232,27 @@ class OrderBook extends EventEmitter {
     return matchingResult;
   }
 
-  private addPeerOrder = (order: orders.PeerOrder) => {
+  private addPeerOrder = (order: orders.PeerOrder): boolean => {
```

Comment:
A comment block would be good here as well to explain that it returns true if the order was added, false if it's a duplicate order or for an invalid pair.